cmd/deployment.go

```diff
@@ -565,6 +581,61 @@ func newDeploymentAirflowUpgradeCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newDeploymentRuntimeRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "runtime",
+		Aliases: []string{"r"},
+		Short:   "Manage runtime deployments",
+		Long:    "Manage runtime deployments",
+	}
+	cmd.AddCommand(
+		newDeploymentRuntimeUpgradeCmd(out),
+		newDeploymentRuntimeMigrateCmd(out),
+	)
+	return cmd
+}
+
+func newDeploymentRuntimeUpgradeCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "upgrade",
+		Aliases: []string{"up"},
+		Short:   "Upgrade Runtime image version",
+		Long:    "Upgrade Runtime image version",
+		Example: deploymentRuntimeUpgradeExample,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return deploymentRuntimeUpgrade(cmd, out)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "d", "", "[ID]")
+	cmd.Flags().StringVarP(&desiredRuntimeVersion, "desired-runtime-version", "v", "", "[DESIRED_AIRFLOW_VERSION]")
```

Comment:
Desired Runtime version you wish to upgrade your deployment to

Reply:
updated